target/snitch_cluster/sw/snax/data-reshuffler/src/snax-data-reshuffler-lib.c

```diff
@@ -80,10 +80,10 @@ uint32_t test_a_chrunk_of_data(int8_t* base_ptr_local, int8_t* base_ptr_l2,
     uint32_t error = 0;
     for (int i = 0; i < len; i++) {
         if ((int8_t)base_ptr_local[i] != (int8_t)base_ptr_l2[i]) {
-            printf(
-                "Error: after reshuffle base_ptr_local[%d] = %d, golden "
-                "base_ptr_l2[%d] = %d \n",
-                i, (int8_t)base_ptr_local[i], i, (int8_t)base_ptr_l2[i]);
+            // printf(
+            //     "Error: after reshuffle base_ptr_local[%d] = %d, golden "
+            //     "base_ptr_l2[%d] = %d \n",
+            //     i, (int8_t)base_ptr_local[i], i, (int8_t)base_ptr_l2[i]);
```

Comment:
Clean or you need this? haha

Reply:
Normally don't need except when bugging :)